### DISCUSSION TITLE
Responsive placeholders

### DISCUSSION THREAD

**Discussion by 13robin37** [UI-Lovelace-Minimalist/UI#115] [Ideas]
Currently some cards still use px units instead of em/rem units and some cards don't use placeholders.

Should we go with a placeholder approach for paddings, margins, fonts and whatnot? I could provide a bunch of units. Might not be the "pixel perfect" solution sometimes but at least all cards will look the same.

**Comment by CM000n** [comment]
So the cards should look different, of course :-D But all joking aside. More consistency in the code is of course always good.
